app/assets/stylesheets/sign_in.scss

```diff
@@ -2,18 +2,13 @@ body#sign-in {
   background: image_url('sign_in_bg.png');
 }
 
-.card-sign-in {
-  .brand-logo {
-    h1 {
-      font-size: 2rem;
-    }
-  }
-}
-
-.card-sign-in-btn {
-  background-color: $secondary-color;
+.container.sign-in {
+  width: 100%;
+  margin-top: 6rem;
 
-  a {
-    width: 100%;
+  h1 {
+    color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`